samples/snippets/src/main/java/com/example/spanner/SpannerSample.java

```diff
@@ -2003,13 +2003,13 @@ static void run(
         observabilityLab.writeMailboxes(dbClient, NUM_MAILBOXES);
     break;
       case "observabilityWriteMessagesInterleavedParallel":
-        observabilityLab.writeMessagesInterleavedParallel(dbClient, NUM_MAILBOXES, MUTATIONS_PER_TRANSACTION*10, NUM_MINUTES*2);
+        observabilityLab.writeMessagesInterleavedParallel(dbClient, NUM_MAILBOXES, MUTATIONS_PER_TRANSACTION*10, NUM_MINUTES*3);
     break;
       case "observabilityLabCreateWorkItems":
         observabilityLab.createWorkItems(dbAdminClient, database);
     break;
       case "observabilityLabWriteWorkItems":
-        observabilityLab.writeWorkItems(dbClient, NUM_ROWS, MUTATIONS_PER_TRANSACTION*10);
+        observabilityLab.writeWorkItems(dbClient, NUM_ROWS*3, MUTATIONS_PER_TRANSACTION*10);
```

Comment:
We can discuss about this number. This will make total workList row to 1k*3=3k. With 10 thread 1K takes around 12-15 min. 
As per Jason's first lab he considered 30vcpu and 8 with second lab, which will be like 8-30 thread. 
Well parallel thread does not speed this work too much but I saw that it reduces the time to 30% less.

Point here is, should we keep 3k or 1k, keeping 3k will atleast run it for 5-10 min with 30 thread.